src/main/java/sg/ncl/MainController.java

```diff
@@ -2440,28 +2442,49 @@ public String adminExperimentsManagement(Model model, HttpSession session) {
         return "experiment_dashboard";
     }
 
-//    @RequestMapping(value = "/admin/{teamId}", method = RequestMethod.GET)
-//    public String admin(@PathVariable String teamId, Model model, HttpSession session) {
-//        HttpEntity<String> exprequest = createHttpEntityHeaderOnly();
-//        ResponseEntity expresponse = restTemplate.exchange(properties.getExpListByTeamId(teamId), HttpMethod.GET, exprequest, String.class);
-//        JSONArray experimentsArray = new JSONArray(expresponse.getBody().toString());
-//
-//        List<Experiment2> experimentList = new ArrayList<>();
-//        Map<Long, Realization> realizationMap = new HashMap<>();
-//
-//        for (int k = 0; k < experimentsArray.length(); k++) {
-//            Experiment2 experiment2 = extractExperiment(experimentsArray.getJSONObject(k).toString());
-//            Realization realization = invokeAndExtractRealization(experiment2.getTeamName(), experiment2.getId());
-//            realizationMap.put(experiment2.getId(), realization);
-//            experimentList.add(experiment2);
-//        }
-//
-//        model.addAttribute("experimentList", experimentList);
-//        model.addAttribute("realizationMap", realizationMap);
-//
-//        return "admin2";
-//    }
+    @RequestMapping("/admin/usage")
+    public String adminTeamUsage(Model model,
+                                 @RequestParam(value = "team", required = false) String team,
+                                 @RequestParam(value = "start", required = false) String start,
+                                 @RequestParam(value = "end", required = false) String end,
+                                 HttpSession session) {
+        if (!validateIfAdmin(session)) {
+            return NO_PERMISSION_PAGE;
+        }
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+        ZonedDateTime now = ZonedDateTime.now();
+        if (start == null) {
+            ZonedDateTime startDate = now.with(firstDayOfMonth());
+            start = startDate.format(formatter);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Introduce a new variable instead of reusing the parameter "start". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)